microbuild-everything.proj

```diff
@@ -0,0 +1,43 @@
+<!-- Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
```

Comment:
This file is not specific to MicroBuild is it?  If not, then change it's name to just ``build-everything.fsproj`` please